### DISCUSSION TITLE
Tablo 4th Gen

### DISCUSSION THREAD

**Discussion by jjtelleria** [jessedp/tablo-tools-electron#212] [General]
I just bought the new 4th gen Tablo OTA DVR.  Will Tablo Tools support this device in the future?  Thanks!

**Comment by jessedp** [comment]
I don't plan on it and based on #210 I'm not sure it's going to be possible. If it is/becomes and I do, I'll update this.

**Comment by Kravimir** [comment]
Since you care about this (missing) feature, I recommend you return it and buy an older model directly from Tablo: https://us-store.tablotv.com/
